WordPress/gutenberg
# Framework: Bump recommended Node version to active LTS

This pull request seeks to increase the recommended version of Node from 6.11.1 to the [active LTS release](https://github.com/nodejs/Release#release-schedule), currently 8.9.0 ([released 2017-10-31](https://nodejs.org/en/blog/release/v8.9.0/)).

__Testing instructions:__

Assuming you use [nvm](https://github.com/creationix/nvm) for managing Node versions, run `nvm install` and `nvm use` to switch to the LTS version. Otherwise install Node to v8.9.0. With a clean node modules (`rm -rf node_modules`) verify that `npm install`, `npm run build`, `npm run dev` complete with no issues.

Verify in Travis CI logs that Node is running 8.9.0 when defined as LTS:

```
3.53s$ nvm install && nvm use
Found '/home/travis/build/WordPress/gutenberg/.nvmrc' with version <lts/*>
Downloading and installing node v8.9.0...
Downloading https://nodejs.org/dist/v8.9.0/node-v8.9.0-linux-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v8.9.0 (npm v5.5.1)
Found '/home/travis/build/WordPress/gutenberg/.nvmrc' with version <lts/*>
Now using node v8.9.0 (npm v5.5.1)
```

https://travis-ci.org/WordPress/gutenberg/jobs/295819466
